service/tbc/tbc.go

```diff
@@ -471,9 +486,19 @@ func (s *Server) localPeerManager(ctx context.Context) error {
 	log.Tracef("localPeerManager")
 	defer log.Tracef("localPeerManager exit")
 
+	if len(s.seeds) != 1 {
+		return fmt.Errorf("expecting 1 seed, received %d", len(s.seeds))
+	}
+
 	peersWanted := 1
 	peerC := make(chan string, peersWanted)
-	address := net.JoinHostPort("127.0.0.1", s.port)
+
+	host, port, err := net.SplitHostPort(s.seeds[0])
+	if err != nil {
+		return err
+	}
+
+	address := net.JoinHostPort(host, port)
```

Comment:
I think this could be simplified to use `s.seeds[0]`, unless there's a reason to split and join?
This could catch an invalid seed, however I think this code would eventually error on an invalid host/port anyway. 🤔

Reply:
yeah we _should_ be able to just use `s.seeds[0]` here, I'll update